color/_tegel_brand_usage.scss

```diff
@@ -0,0 +1,292 @@
+  --foreground-border-inverse-defined: var(--tds2-neutral-transparent-inverse-700);
+  --foreground-border-inverse-soft: var(--tds2-neutral-transparent-inverse-600);
+  --foreground-border-inverse-subtle: var(--tds2-neutral-transparent-inverse-500);
+  --foreground-border-inverse-disabled: var(--tds2-neutral-transparent-inverse-400);
+  --foreground-border-inverse-discrete: var(--tds2-neutral-transparent-inverse-200);
+
+  // Foreground - Border - Clickable
+  --foreground-border-clickable-link: var(--tds2-blue-400);
+  --foreground-border-clickable-focus: var(--tds2-blue-400);
+
+  //TODO: Migrate to unit tokens
+  --foreground-border-clickable-focus-width: 2px;
+
+  // Foreground - Elements - Transparent
+  --foreground-elements-transparparent-strong: var(--tds2-neutral-transparent-958);
+  --foreground-elements-transparparent-defined: var(--tds2-neutral-transparent-700);
+  --foreground-elements-transparparent-soft: var(--tds2-neutral-transparent-600);
+  --foreground-elements-transparparent-subtle: var(--tds2-neutral-transparent-500);
+  --foreground-elements-transparparent-disabled: var(--tds2-neutral-transparent-400);
+  --foreground-elements-transparparent-discrete: var(--tds2-neutral-transparent-200);
+  --foreground-elements-transparparent-strong-inverse: var(--tds2-neutral-transparent-inverse-958);
+  --foreground-elements-transparparent-defined-inverse: var(--tds2-neutral-transparent-inverse-700);
+  --foreground-elements-transparparent-soft-inverse: var(--tds2-neutral-transparent-inverse-600);
+  --foreground-elements-transparparent-subtle-inverse: var(--tds2-neutral-transparent-inverse-500);
+  --foreground-elements-transparparent-disabled-inverse: var(
+    --tds2-neutral-transparent-inverse-400
+  );
+  --foreground-elements-transparparent-discrete-inverse: var(
+    --tds2-neutral-transparent-inverse-200
+  );
+
+  // Foreground - Elements - Solid
+  --foreground-elements-solid-1000: var(--tds2-neutral-solid-958);
+  --foreground-elements-solid-900: var(--tds2-neutral-solid-800);
+  --foreground-elements-solid-800: var(--tds2-neutral-solid-700);
+  --foreground-elements-solid-700: var(--tds2-neutral-solid-600);
+  --foreground-elements-solid-600: var(--tds2-neutral-solid-500);
+  --foreground-elements-solid-500: var(--tds2-neutral-solid-400);
+  --foreground-elements-solid-400: var(--tds2-neutral-solid-300);
+  --foreground-elements-solid-300: var(--tds2-neutral-solid-200);
+  --foreground-elements-solid-200: var(--tds2-neutral-solid-100);
+  --foreground-elements-solid-100: var(--tds2-neutral-solid-50);
+
+  // Background - Elevation
+  --background-elevation-base: var(--tds2-neutral-solid-00);
+  --background-elevation-layer-01: var(--tds2-neutral-solid-50);
+  --background-elevation-layer-02: var(--tds2-neutral-solid-00);
+  --background-elevation-layer-03: var(--tds2-neutral-solid-50);
+  --background-elevation-top: var(--tds2-neutral-solid-00);
+  --background-elevation-scrim: rgb(0 0 0 / 38%);
```

Comment:
What to do with this? And its name, should it be overlay?

Reply:
If this will be used for modal/dialogs, I think the name should include overlay/backdrop